webapp/app/data_access/storage/form_storage.py

```diff
@@ -0,0 +1,57 @@
+
+import logging
+import zlib
+
+from cryptography.fernet import InvalidToken
+from flask import json
+from typing import Optional
+from abc import ABC, abstractmethod
+
+from app.crypto.encryption import encrypt, decrypt
+
+logger = logging.getLogger(__name__)
+
+class FormStorage(ABC):
+
+    @abstractmethod
+    def get_data(self, data_identifier, ttl: Optional[int] = None, default_data=None):
+        """
+        Gets data from a storage that is associated with the currect session. The data is return deserialized.
+
+        :param data_identifier: A string used to identify what data should be taken from the cookie
+        :param ttl: The time to live for the cookie
+        :param default_data: Default data that will be used to replace missing data points
+        """
+        pass
+
+    @abstractmethod
+    def override_data(self, data_to_store, data_identifier):
```

Comment:
Why don't we use static methods or class methods for these, too? We would not need to initialize the class in the steps

Reply:
Done